WindowsServerDocs/identity/ad-cs/create-domain-user-account-as-NDES-service-account.md

```diff
@@ -0,0 +1,179 @@
+
+Applies To: Windows Server (All supported versions)
+
+In this article you will learn how to configure NDES to run as a user account that is specified as a service account.
+
+The Network Device Enrollment Service (NDES) allows software on routers and other network devices running without domain credentials to obtain certificates based on the Simple Certificate Enrollment Protocol (SCEP).
+
+> [!NOTE]
+> If you select the built-in application pool identity, there is no additional configuration required.
+
+The recommended configuration is to specify a user account, which requires additional steps.
+
+SCEP was developed to support the secure, scalable issuance of certificates to network devices by using existing certification authorities (CAs). The protocol supports CA and registration authority public key distribution, certificate enrollment, certificate revocation, certificate queries, and certificate revocation queries.
+
+See [What is Network Device Enrollment Service for Active Directory Certificate Services?](/network-device-enrollment-service-overview.md) to learn more about NDES and how it works with certificates based on Simple Certificate Enrollment Protocol.
+
+## Specify a user account as a service account in NDES
+
+As stated in the preceding section, the recommended configuration for NDES is to specify a user account and that this requires additional steps to complete.
+
+## Prerequisites
+
+After installing the NDES role service for AD CS, verify that the following prerequisites are satisfied:
+
+- Be a domain user account
+
+- Be a member of the local IIS\_IUSRS group
+
+- Have Request permissions on the configured CA
+
+- Have Read and Enroll permissions on the NDES certificate template, which is configured automatically
+
+## Verify whether it is necessary to set a service principal name for NDES
+
+You need to configure a service principal name (SPN) in Active Directory if you're using a load balancer or virtual name. In this section you'll learn how to  determine whether it is necessary to set an SPN in Active Directory.
+
+- If you are using a single NDES server and its actual hostname (most common scenario), the account does not need an SPN registered. The computer accounts default SPNs for HOST/computerfqdn cover this case. If you are using all other defaults (particularly around IIS kernel-mode authentication), you can skip ahead to the next section of this article.
+
+- If you are using a custom A record as a hostname, or load balancing with a Virtual IP, an SPN needs to be registered against the NDES service account (SCEPSvc). To register an SPN against the NDES service account:
+ 
+  1. Use the Setspn command syntax of: **Setspn -s HTTP/computerfqdn** domainname\accountname when entering your commands. For example:
+
+    - **Setspn -s HTTP/NDESFARM.fabrikam.com fabrikam\SCEPSvc**
+    - **Setspn -s HTTP/NDESFARM fabrikam\SCEPSvc**
+
+  1. Then disable IIS Kernel-mode Authentication for the site.
```

Comment:
This is duplicated with line 119 - 125, I'm thinking it would be best to merge and likely reposition this section to line 119. My thinking is, how can we set the SPN yet when we haven't yet created the account?